app/views/items/_search_res_items.html.erb

```diff
@@ -1,4 +1,4 @@
-<% if @res.count > 0 %>
+<% if @res.count && @res.count > 0 %>
```

Comment:
I think `@res.present?` will work in this case. But the other with `@res.count > 1` wouldn't because that's different logic.

Reply:
Oh yeah I totally meant to do `@res && @res.count`